doc/manual/config-advanced.rst

```diff
@@ -32,6 +32,20 @@ path of your installation) as follows:
   The IDs for affiliations and teams need to be the *external ID*
   if the ``data_source`` setting of DOMjudge is set to external.
 
+It is also possible to load custom CSS and/or JavaScript files. To do so, place
+files ending in `.css` under `public/css/custom/` and/or files ending in `.js`
```

Comment:
` must be `` throughout this paragraph

Reply:
No? Well at least it is also not the case for all paragraphs above, only for configuration settings.

Reply:
It seems we’re not very consistent in the manual with this. So not an issue for this PR then